pages/science/reactors/cvodes.rst

```diff
@@ -0,0 +1,242 @@
+  exactly the times specified. Internally, several ``step()`` calls are
+  typically performed to reach the accurate state at time
+  :math:`t_{\mathrm{new}}`.
+
+- ``advance_to_steady_state(max_steps, residual_threshold, atol,
+  write_residuals)`` [Python interface only]: If the steady state solution of a
+  reactor network is of interest, this method can be used. Internally, the
+  steady state is approached by time stepping. The network is considered to be
+  at steady state if the feature-scaled residual of the state vector is below a
+  given threshold value (which by default is 10 times the time step ``rtol``).
+
+The use of the ``advance`` method in a loop has the advantage that it produces
+results corresponding to a predefined time series. These are associated with a
+predefined memory consumption and well comparable between simulation runs with
+different parameters. However, some detail (for example, a fast ignition process)
+might not be resolved in the output data due to the typically large time steps.
+To avoid losing this detail, the
+`Reactor::setAdvanceLimit <{{% ct_docs doxygen/html/dc/d5e/classCantera_1_1Reactor.html#a9b630edc7d836e901886d7fd81134d9e %}}>`__
+method (C++) or the :py:func:`Reactor.set_advance_limit` method (Python) can be
+used to set the maximum amount that a specified solution component can change
+between output times. For an example of this feature's use, see the example
+`reactor1.py </examples/python/reactors/reactor1.py.html>`__.
+
+The ``step`` method results in many more data points because of the small
+timesteps needed. Additionally, the absolute time has to be kept track of
+manually.
+
+Even though Cantera comes pre-defined with typical parameters for tolerances
+and the maximum internal time step, the solution sometimes diverges. To solve
+this problem, three parameters can be tuned: The absolute time stepping
+tolerances, the relative time stepping tolerances, and the maximum time step. A
+reduction of the latter value is particularly useful when dealing with abrupt
+changes in the boundary conditions (for example, opening/closing valves; see
+also the `IC engine example </examples/python/reactors/ic_engine.py.html>`__).
+
+How Does Cantera's Reactor Network Time Integration Feature Actually Work?
+==========================================================================
+
+There's a great description of the science behind Cantera's reactor network 
+simulation capabilities available on the Cantera website, 
+`here <https://cantera.org/science/reactors/reactors.html>`__. This section will go into more 
+developer-oriented detail about how the last step, `ReactorNet`'s 
+`time integration methods <https://cantera.org/science/reactors/reactors.html#time-
+integration-for-reactor-networks>`__, actually work. A `ReactorNet` object doesn't 
+perform time integration on its own. It generates a system of ODE's based on the 
```

Comment:
```suggestion
perform time integration on its own. It generates a system of ODEs based on the 
```

What does it mean to "generate" a system of ODEs in this context? Cantera never actually writes down any of the equations from the `reactors.html` page... It's probably more accurate to say something more like, Cantera generates a Jacobian array for the set of Reactors installed in the ReactorNet. This Jacobian, along with functions to evaluate the residual of each ODE representing the system, is passed to an Integrator. You'll need to tweak the phrasing there probably, but I think that's somewhat better 😄